ggml-cuda.cu

```diff
@@ -442,10 +479,17 @@ void * ggml_cuda_host_malloc(size_t size) {
     void * ptr = nullptr;
     cudaError_t err = cudaMallocHost((void **) &ptr, size);
     if (err != cudaSuccess) {
+        // The allocation error can be bypassed. A null ptr will assigned out of this function.
+        // This can fixed the OOM error in WSL.
+        cudaGetLastError();
```

Comment:
why call this function here? You already have err?

Reply:
Failed to allocate memory error.

Reply:
sorry, i mean why we need to call cudaGetLastError and also discard the return code?

Reply:
Yes, just ignore the cuda error. In llama_ctx_buffer.resize(), it will fall back to pageable memory if it's encountering errors.